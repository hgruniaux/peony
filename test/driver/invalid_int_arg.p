//# RUN: peony -fmax-errors=5 -fmax-errors=hello -fdiagnostics-column-origin=5h
//~! ERROR: argument to '-fmax-errors' should be an integer
//~! ERROR: argument to '-fdiagnostics-column-origin' should be an integer
