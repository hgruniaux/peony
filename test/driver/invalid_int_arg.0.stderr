error: argument to '-fmax-errors' should be an integer
error: argument to '-fdiagnostics-column-origin' should be an integer
